lib/tasks/backfill_gpo_expiration.rake

```diff
@@ -0,0 +1,64 @@
+# rubocop:disable Metrics/BlockLength
+namespace :profiles do
+  desc 'Backfill the gpo_verification_expired_at value'
+
+  ##
+  # Usage:
+  #
+  # Print pending updates
+  # bundle exec rake profiles:backfill_gpo_expiration > profiles.csv
+  #
+  # Commit updates
+  # bundle exec rake profiles:backfill_gpo_expiration UPDATE_PROFILES=true > profiles.csv
+  #
+  task backfill_gpo_expiration: :environment do |_task, _args|
+    min_profile_age = (ENV['MIN_PROFILE_AGE_IN_DAYS'].to_i || 100).days
+    update_profiles = ENV['UPDATE_PROFILES'] == 'true'
+
+    job = GpoExpirationJob.new
+
+    profiles = job.gpo_profiles_that_should_be_expired(
+      as_of: Time.zone.now,
```

Comment:
I thought I saw an additional commit changing `as_of:` to `now:` in the rest of the code, but I still see `as_of:` elsewhere. Do you want to change it everywhere?

Reply:
I think doing it as `now` for the Job args makes sense, but I think `as_of` fits better with the fluent style of the other methods. Mostly a matter of taste, I guess.